### ISSUE TITLE
In the emote picker, the section for 7TV channel emotes should be called <StreamerName>'s Emotes, just like for the Kick channel emotes

### ISSUE THREAD

**Issue by RevSnowfox** [Xzensi/NipahTV#139]
Ideally using the same form of the username as the Kick emote section. (I don't know if NTV could fetch a username from 7TV, but best to use the capitalization found on Kick itself.)

**Comment by Xzensi** [comment]
Agreed, considering 7TV only allows one active emote set per plaform connection, it'll make emote set names more consistent.
